css/grid.css

```diff
@@ -0,0 +1,28 @@
+.container{
+    min-height: 100vh;
+    display: grid;
+    gap: 3px;
+    grid-template-rows: auto auto auto 1fr;
```

Comment:
To me, feels like there is no need of using `grid` here, because you are just stacking element in top of each other, so for that purpose you can just use `flex` or even other properties without `flex`. Grid could be use when you need to organize your elements in `columns` and `rows`, controlling the spacing, position etc, when you need to control your layout more precisely.